e2e_tests/e2e_test.py

```diff
@@ -0,0 +1,78 @@
+import os
+from threading import Thread
+from trello.trello_api import TrelloAPI
+import app
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import time
+
+@pytest.fixture(scope='module')
+def test_app():
+    # Create the new board & update the board id environment variable
+    trello_board = TrelloAPI().create_board("SeleniumTest")
+    os.environ['TRELLO_BOARD_ID'] = trello_board.board_id
+    # construct the new application
+    application = app.create_app()
+    # start the app in its own thread.
+    thread = Thread(target=lambda: application.run(use_reloader=False))

```

Comment:
FYI this will fail if load_dotenv is called from a previous test (to fix this you'd need to call `load_dotenv(".env", override=True)`)

Reply:
This won't prevent this PR from passing as you've documented how to run the tests separately 👍